components/layout_2020/style_ext.rs

```diff
@@ -493,7 +509,8 @@ impl ComputedValuesExt for ComputedValues {
         // <https://drafts.csswg.org/css-transforms/#transformable-element>
         // TODO: check for all cases listed in the above spec.
         !self.get_box().display.is_inline_flow() ||
-            fragment_flags.contains(FragmentFlags::IS_REPLACED)
+            fragment_flags.contains(FragmentFlags::IS_REPLACED) ||
+            fragment_flags.contains(FragmentFlags::IS_INPUT_ELEMENT)
```

Comment:
This should also be checked in `BoxFragment::is_inline_box()`. Otherwise the `clientWidth` will be 0 for `input style="display: inline">`.

It may be better to add here in `style_ext.rs` a `pub(crate) fn is_inline_box(&self, fragment_flags: FragmentFlags) -> bool` method to be called from both `is_transformable()` and `BoxFragment::is_inline_box()`.